### PR TITLE
Add a command for easy install of the 'helm' tool.

### PR DESCRIPTION
**Pull request by brendandburns**
@seanknox

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -281,9 +281,71 @@ def dcos_install_cli(install_location=None, client_version='1.8'):
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
 
+def k8s_install_draft(client_version='v0.5.1', install_location=None):
+    base_url = 'https://github.com/Azure/draft/releases/download/{}/draft-{}-{}-amd64.tar.gz'
```
Comment (troydai) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: This is not a good place to store a file which will be public consumed. The github release will throttle the download. Visual Studio Code once run into this issue when they pointed an extension download directly to a github release download link.

Use Azure storage blob to stage this file.

Comment (seanknox) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: @brendandburns Let me know if you want me to handle setting up blob storage for Helm.

**Comment by troydai** [comment]
Update?

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (seanknox) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: I'll get blob storage setup for this.

Comment (bacongobbler) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: We already have releases pushed to WABS.

https://azuredraft.blob.core.windows.net/draft/draft-v0.6.0-linux-amd64.tar.gz

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -281,9 +281,71 @@ def dcos_install_cli(install_location=None, client_version='1.8'):
         raise CLIError('Connection error while attempting to download client ({})'.format(err))
 
 
+def k8s_install_draft(client_version='v0.5.1', install_location=None):
```
Comment (bacongobbler) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: this should be bumped to v0.6.0

**Comment by seanknox** [comment]
@brendandburns can you grant me commit access to your fork?

**Comment by troydai** [comment]
@seanknox, alternatively you can just pull this branch, make the change, and submit another PR. I'm happy to review it.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 2 above)
Comment (brendandburns) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: done.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py (hunk 1 above)
Comment (brendandburns) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: changed draft. @seanknox sounds great, if you send me a URL I'll update (I also granted you access...)

**Comment by derekbekoe** [comment]
Closing as this PR has been open for a while and has conflicts.
It should also be based off dev instead of master.

**Pull request closed without merging**